index.html

```diff
@@ -10,185 +10,225 @@
-        </li>
-      </ul>
-    </section>
-  </div>
-    <div class="work-section">
-        <div class="container container1">
-            <div class="Card1"><img src="images/img1.png" alt="" class="card1"></div>
-            <div>
-                <h1>Tonic</h1>
-            </div>
-            <div class="lister">
-                <p><strong>CANOPY</strong></p>
-                <div class="point"></div>
-                <P>Back End Dev</P>
-                <div class="point"></div>
-                <p>2015</p>
-            </div>
+    <div class="main-wrapper">
+        <div class="wrapper">
+            <header class="toolbar">
+                <div class="right-side">
+                    <div class="logo"><a href="#">My logo</a></div>
+                </div>
+
+                <nav class="desktop-nav">
+                    <ul>
+                        <li><a href="#">Portfolio</a></li>
+                        <li><a href="#">About</a></li>
+                        <li><a href="#">Contact</a></li>
+                    </ul>
+                </nav>
 
-            <p id="para1">A daily selection of privately <br>personalized roads; no accounts or <br>sign-up required.
-            </p>
-            <ul>
-                <li id="html"><a href="#"> html</a></li>
-                <li id="css"><a href="#">css</a></li>
-                <li id="javaScript"><a href="#">javaScript</a></li>
-            </ul>
-            <div class="to-project"><a href="#">See Project</a></div>
+                <nav>
+                    <div class="menu-line"></div>
+                    <div class="menu-line"></div>
+                    <div class="menu-line"></div>
+                </nav>
+            </header>
+            <section class="head-lines">
+                <h1>I'm Anicet<br />Glad to see <br />you!</h1>
+                <p>
+                    I'm a software developer I can help you build a product, feature
+
+                    website Look through some of my work
+                    and experience! if you see
+                    and have a project you need coded.
+                    don't hestiate to contact me.
+                </p>
+                <div class="connect"><a href="">LET'S CONNECT</a></div>
+                <ul>
+                    <li>
+                        <a href=""><img src="./images/twiter.png" /></a>
+                    </li>
+                    <li>
+                        <a href=""><img src="./images/linkedin.jpeg" /></a>
+                    </li>
+                    <li>
+                        <a href=""> <img src="./images/Muliti.png" /></a>
+                    </li>
+                    <li>
+                        <a href=""><img src="./images/github.png" /></a>
+                    </li>
+                    <li>
+                        <a href=""><img src="./images/sign.png" /></a>
+                    </li>
+                </ul>
+            </section>
```

Comment:
Kindly try to match the `background-image` for the `hero` section with the Figma design in desktop version.
| Your design | Figma design |
| --- | --- |
| ![image](https://user-images.githubusercontent.com/24703171/155523902-f03e3a60-fe3d-453e-a266-7baf9222c533.png) | ![Template 2-Main (1) - Copy](https://user-images.githubusercontent.com/24703171/155524255-26472be4-91c7-453a-8001-9211248d732e.png) |